packages/lib-react-components/src/Tab/Tab.spec.js

```diff
@@ -1,42 +1,72 @@
+import { Key } from 'grommet-icons'
 
 import Tabs from '../Tabs'
 import Tab from './Tab'
 import SpacedText from '../SpacedText'
 
-describe('<Tab />', function () {
-  it('should render without crashing', function () {
-    const wrapper = shallow(<Tab title='Foobar' />)
-    expect(wrapper).to.be.ok()
-  })
+describe('Component > Tab', function () {
+  describe('with standard props', function () {
+    beforeEach(function () {
+      render(
+        <Grommet theme={zooTheme} >
+          <Tabs>
+            <Tab title='apples'>An apple is a red fruit.</Tab>
+            <Tab title='bananas'>A banana is a long fruit.</Tab>
+            <Tab title='cherries'>A cherry is a stone fruit.</Tab>
+          </Tabs>
+        </Grommet>
+      )
+    })
+
+    it('should render without crashing', function () {
+      const tabsContainer = screen.getByRole('tablist')
+      const arrayOfTabs = screen.getAllByRole('tab')
+      const tabPanel = screen.getByRole('tabpanel')
+      expect(tabsContainer).to.exist()
+      expect(arrayOfTabs).to.have.length(3)
+      expect(tabPanel).to.exist()
+    })
+
+    it('should render the `title` prop as standard text if it\'s a string', function () {
+      const arrayOfTabs = screen.getAllByRole('tab')
 
-  it('should render the `title` prop using the SpacedText component if it\'s a string', function () {
-    const TITLE = 'Foobar'
-    const wrapper = mount(
-      <Grommet theme={zooTheme}>
-        <Tabs>
-          <Tab title={TITLE} />
-        </Tabs>
-      </Grommet>
-    )
-    const tabTitle = wrapper.find(SpacedText)
-    expect(tabTitle).to.have.lengthOf(1)
-    expect(tabTitle.text()).to.equal(TITLE)
+      const tab0 = within(arrayOfTabs[0])
+      const tab1 = within(arrayOfTabs[1])
+      const tab2 = within(arrayOfTabs[2])
+
+      expect(tab0.getByText('apples')).to.exist()
+      expect(tab1.getByText('bananas')).to.exist()
+      expect(tab2.getByText('cherries')).to.exist()
+    })
   })
 
-  it('should should pass through the `title` prop if it\'s a component', function () {
-    const TITLE = () => (<div>Foobar</div>)
-    const wrapper = mount(
-      <Grommet theme={zooTheme}>
-        <Tabs>
-          <Tab title={<TITLE />} />
-        </Tabs>
-      </Grommet>
-    )
-    const tabTitle = wrapper.find(TITLE)
-    expect(tabTitle).to.have.lengthOf(1)
+  describe('with complex props', function () {
+    it('should should pass through the `title` prop if it\'s a component', function () {
+      const ComplexTitle = () => (
+        <div>
+          <Key size='small' />
+          &nbsp;
+          <b data-testid="b-tag">Special</b>
+        </div>
+      )
+
+      render(
+        <Grommet theme={zooTheme} >
+          <Tabs>
+            <Tab title={<ComplexTitle />}>A tab with a rather complex title (tab header)</Tab>
+            <Tab title='bananas'>A banana is a long fruit.</Tab>
+          </Tabs>
+        </Grommet>
+      )
+
+      const keySvg = screen.getByLabelText('Key')
```

Comment:
```suggestion
      const complexTab = screen.getByLabelText('Key Special')
      const keySvg = screen.getByLabelText('Key')
```
Maybe add a query that checks the name of the tab in screen readers? I'm not convinced that 'Key Special' is an accessible name, to be honest.